terraform/modules/database/variables.tf

```diff
@@ -1,4 +1,4 @@
-variable "pgbouncer_subnet_id" {
+gvariable "pgbouncer_subnet_id" {
```

Comment:
Just fix this typo, then I think you're good to go!

Reply:
ahh how did that sneak in there